torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1757,6 +1757,7 @@
   name: eye
   backends:
     - CPU
+    - CUDA
```

Comment:
@fmassa I think that would be good in theory, but in practice for now I'm not sure if there is any push to do so. I'm still in the progress of migrating THD to ATen, which could make that process easier (ATen having consistent semantics for passing parameters, while THPP does not).